nexus-repository-r/src/test/java/org/sonatype/nexus/repository/r/internal/upload/RUploadHandlerTest.java

```diff
@@ -155,10 +157,39 @@ public void testHandle_unauthorized() throws IOException {
     }
   }
 
-  private ComponentUpload createComponentUpload(final String packagePath) {
+  @Test
```

Comment:
looks like it will be better to create a parametrized unit test to test a set of paths

Reply:
done in c4a197a630ce727a180161939665cd61193700a9